mpmp/scripts/run_tumor_purity.sh

```diff
@@ -0,0 +1,20 @@
+#!/bin/bash
```

Comment:
Maybe also here a short description of this script, that I think uses compressed and non-compressed data.